nautobot/circuits/tests/integration/test_relationships.py

```diff
@@ -0,0 +1,114 @@
+            slug="test-circuit-type",
+        )
+        circuit = Circuit.objects.create(
+            provider=provider1,
+            cid="1234",
+            type=circuit_type,
+            status=active_circuit_status,
+        )
+        site = Site.objects.create(
+            name="Test Site",
+            slug="test-site",
+            status=active_site_status,
+        )
+        circuit_termination = CircuitTermination.objects.create(
+            circuit=circuit,
+            term_side="A",
+            site=site,
+        )
+        power_panel = PowerPanel.objects.create(
+            site=site,
+            name="Test Power Panel",
+        )
+        m2m = Relationship.objects.create(
+            name="Termination 2 Provider m2m",
+            slug="termination-2-provider-m2m",
+            source_type=circuit_termination_ct,
+            destination_type=provider_ct,
+            type=RelationshipTypeChoices.TYPE_MANY_TO_MANY,
+        )
+        RelationshipAssociation.objects.create(
+            relationship=m2m,
+            source=circuit_termination,
+            destination=provider1,
+        )
+        RelationshipAssociation.objects.create(
+            relationship=m2m,
+            source=circuit_termination,
+            destination=provider2,
+        )
+        o2m = Relationship.objects.create(
+            name="Termination 2 Site o2m",
+            slug="termination-2-provider-o2m",
+            source_type=circuit_termination_ct,
+            destination_type=site_ct,
+            type=RelationshipTypeChoices.TYPE_ONE_TO_MANY,
+        )
+        RelationshipAssociation.objects.create(
+            relationship=o2m,
+            source=circuit_termination,
+            destination=site,
+        )
+        o2o = Relationship.objects.create(
+            name="Termination 2 Power Panel o2o",
+            slug="termination-2-power-panel-o2o",
+            source_type=circuit_termination_ct,
+            destination_type=power_panel_ct,
+            type=RelationshipTypeChoices.TYPE_ONE_TO_ONE,
+        )
+        RelationshipAssociation.objects.create(
+            relationship=o2o,
+            source=circuit_termination,
+            destination=power_panel,
+        )
+
+    def tearDown(self):
+        self.logout()
+        super().tearDown()
+
+    def test_relationships_are_visible(self):
+        """
+        Navigate to the circuit created in setUp() and check that the relationships are showing on the page
+        """
+        self.browser.visit(self.live_server_url)
+        self.browser.links.find_by_partial_text("Circuits")[0].click()
+        self.browser.links.find_by_partial_text("Circuits")[1].click()
+        self.browser.links.find_by_partial_text("1234").click()
+        self.assertTrue(self.browser.is_text_present("Power Panel"))
+        self.assertTrue(self.browser.is_text_present("2 providers"))
+        self.assertTrue(self.browser.is_text_present("1 site"))
```

Comment:
Some comments here to explain what expected behavior is being tested would be helpful - I think I was able to figure it out from context but it's not immediately obvious.